server/queue/queue-activemq/pom.xml

```diff
@@ -35,9 +35,11 @@
     <dependencies>
         <!-- Forcing netty-transport version -->
         <dependency>
+            <?SORTPOM IGNORE?>
```

Comment:
why?

Reply:
Prevent a reordering causing netty override to be lost i guess

Reply:
exactly this is one of the few cases where dependency declaration order is important more information at https://github.com/Ekryd/sortpom/wiki/Parameters-that-can-affect-your-build